src/codemodder/codemods/utils_mixin.py

```diff
@@ -58,6 +63,44 @@ def _is_direct_call_from_imported_module(
                             return (import_node, alias)
         return None
 
+    def get_imported_prefix(
+        self, node
+    ) -> Optional[tuple[Union[cst.Import, cst.ImportFrom], cst.ImportAlias]]:
+        """
+        Given a node representing an access, finds if any part of its prefix is imported.
+        Returns a import and import alias pair.
+        """
+        for nodo in iterate_left_expressions(node):
+            match nodo:
+                case cst.Name() | cst.Attribute():
+                    maybe_assignment = self.find_single_assignment(nodo)
+                    if maybe_assignment and isinstance(
+                        maybe_assignment, ImportAssignment
+                    ):
+                        import_node = maybe_assignment.node
+                        for alias in import_node.names:
+                            if maybe_assignment.name in (
+                                alias.evaluated_alias,
+                                alias.evaluated_name,
+                            ):
+                                return (import_node, alias)
+        return None
```

Comment:
surprised pylint nor mypy didn't complain but all returns of a func should return the same len, so this should be (None, None)